Bugzilla/Auth.pm

```diff
@@ -111,6 +111,26 @@ sub login {
         });
     }
 
+    my $mfa_group = 'admin';
+    my %mfa_allowed_page = map { $_ => 1 } qw(index.cgi post_bug.cgi process_bug.cgi userprefs.cgi);
```

Comment:
This is a lot of work to support and opens some interesting MFA-bypass opportunities through process_bug.cgi. I think you'd be better off taking BMO offline for maintenance during a scheduled window and shipping this as a breaking change, without any exceptions other than `userprefs.cgi?tab=mfa` only.